### ISSUE TITLE
incorrectly exact trigonometric results in v13.2

### ISSUE THREAD

**Issue by jgoizueta** [numworks/epsilon#1558]
#### Describe the bug

In radians mode, some trigonometric results near to 0 are incorrectly rounded to 0 in v13.2; (e.g. `sin(3.1415926535898)`). This didn't happen in versions up to 12.4.

#### Screenshots

Version <= 12.4 (correct result):

![screenshot](https://user-images.githubusercontent.com/5909/82575837-45143b00-9b89-11ea-9708-7ccadfe58334.png)

Version 13.2 (incorrect):

![screenshot_1](https://user-images.githubusercontent.com/5909/82575930-670dbd80-9b89-11ea-87d0-1b8f3e60933f.png)

#### To Reproduce
1. Go to the Calculation app (having set angle mode to radians)
2. Type `sin(3.1415926535898)`
3. Result is (exact) 0.

#### Expected behavior

Result in the previous example should be  -6.9829626776863E-15

#### Environment
 - Epsilon version 13.2.0
 - Tested in actual device and simulators

#### Additional information

I've been playing a little with the code and this seems to fix the issue:

```diff
diff --git a/poincare/src/cosine.cpp b/poincare/src/cosine.cpp
index 2e216ce8e..0098d5a8e 100644
--- a/poincare/src/cosine.cpp
+++ b/poincare/src/cosine.cpp
@@ -19,7 +19,7 @@ template<typename T>
 Complex<T> CosineNode::computeOnComplex(const std::complex<T> c, Preferences::ComplexFormat, Preferences::AngleUnit angleUnit) {
   std::complex<T> angleInput = Trigonometry::ConvertToRadian(c, angleUnit);
   std::complex<T> res = std::cos(angleInput);
-  return Complex<T>::Builder(ApproximationHelper::NeglectRealOrImaginaryPartIfNeglectable(res, angleInput));
+  return Complex<T>::Builder(res);
 }

 Layout CosineNode::createLayout(Preferences::PrintFloatMode floatDisplayMode, int numberOfSignificantDigits) const {
diff --git a/poincare/src/sine.cpp b/poincare/src/sine.cpp
index 3668d8a3a..9a0ec2f46 100644
--- a/poincare/src/sine.cpp
+++ b/poincare/src/sine.cpp
@@ -19,7 +19,7 @@ template<typename T>
 Complex<T> SineNode::computeOnComplex(const std::complex<T> c, Preferences::ComplexFormat, Preferences::AngleUnit angleUnit) {
   std::complex<T> angleInput = Trigonometry::ConvertToRadian(c, angleUnit);
   std::complex<T> res = std::sin(angleInput);
-  return Complex<T>::Builder(ApproximationHelper::NeglectRealOrImaginaryPartIfNeglectable(res, angleInput));
+  return Complex<T>::Builder(res);
 }
```

Bear in mind I have very little understanding of the code base, I haven't done any significant amount of testing and I don't really know what I'm doing 😛 I don't mean to suggest this is a proper fix for this problem.

**Comment by adriweb** [comment]
This seems to be explicitly desired behaviour: https://github.com/numworks/epsilon/commit/a2e9d8efd6357d79c588b66a78a5bc8888744704#diff-d27313652263a0a25461c5c3d412b9a8R39-R47

**Comment by artaxxx** [comment]
Fixed in 15.0.0
